src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Runtime/Installation/FileBackedPluginsInstaller.cs

```diff
@@ -369,9 +370,9 @@ public async Task CleanupObsoletePluginsAsync(CancellationToken cancellationToke
             string installPath = PathUtils.GetPluginInstallDirectory(
                 this.installationRoot,
                 new PluginIdentity(installedPlugin.Id, installedPlugin.Version));
-            
-            string metadataFileName = Path.Combine(installPath, PluginPackage.PluginMetadataFileName);
-            
+
+            string metadataFileName = Path.Combine(installPath, ZipPluginPackage.PluginMetadataFileName);
```

Comment:
PluginMetadataFileName should probably be defined in this class now

Reply:
As in, the filename for where we install the metadata file :)

Reply:
Have moved the names to a constants file.  Going to have another PR for moving things like reading metadata to `storage`